webapp/src/Controller/API/GeneralInfoController.php

```diff
@@ -327,6 +333,9 @@ public function countryFlagAction(Request $request, string $countryCode, string
         if (!Countries::alpha3CodeExists($alpha3code)) {
             throw new NotFoundHttpException("country $alpha3code does not exist");
         }
+        if (!preg_match('/^\d+x\d+$/', $size)) {
+            throw new BadRequestHttpException('invalid format for size parameter');
```

Comment:
nit: can we be more specific here and say that we would expect `<int>x<int>, e.g. 64x64`?

Reply:
I've gone with just making the two supported values explicit. We can do al kinds of dir reading in here to determine what flag aspects are there but that seems overkill to me, likely this will not change often.